R/users.R

```diff
@@ -179,3 +179,22 @@ users_exist <- function(vault, users) {
 on_failure(users_exist) <- function(call, env) {
   paste0("Secret ", deparse(call$users), " do not exist")
 }
+
+#' @importFrom openssl fingerprint read_pubkey
+
+lookup_user <- function(key, vault) {
+  if (is.character(key)) {
+    key <- tryCatch(read_key(key),
+                    error = function(e) NULL)
```

Comment:
Can you pls write this in one line, or like this:
```r
key <- tryCatch(
  read_key(key),
  error = function(e) NULL
)
```
To match the style of the existing code. Thanks.